lib/logstash/outputs/rabbitmq.rb

```diff
@@ -15,6 +18,8 @@ class RabbitMQ < LogStash::Outputs::Base
       include LogStash::PluginMixins::RabbitMQConnection
 
       config_name "rabbitmq"
+      
+      default :codec, "json"
```

Comment:
Should that read more like:

```
  config :codec, :default => "json"
```

See the [file input](https://github.com/logstash-plugins/logstash-input-file/blob/master/lib/logstash/inputs/file.rb#L67-L69) for an example.